actions/contacts/routes.go

```diff
@@ -12,21 +12,45 @@ type Action struct {
 	actions.Action
 }
 
-// NewHandler creates the specific package routes
-func NewHandler(appConfig *config.AppConfig, services *config.AppServices) routes.OldAPIEndpointsFunc {
+// OldContactsHandler creates the specific package routes
+func OldContactsHandler(appConfig *config.AppConfig, services *config.AppServices) routes.OldAPIEndpointsFunc {
 	action := &Action{actions.Action{AppConfig: appConfig, Services: services}}
 
 	apiEndpoints := routes.OldAPIEndpointsFunc(func(router *gin.RouterGroup) {
 		group := router.Group("/contact")
-		group.PUT("/:paymail", action.upsert)
+		group.PUT("/:paymail", action.oldUpsert)
 
-		group.PATCH("/accepted/:paymail", action.accept)
-		group.PATCH("/rejected/:paymail", action.reject)
-		group.PATCH("/confirmed/:paymail", action.confirm)
-		group.PATCH("/unconfirmed/:paymail", action.unconfirm)
+		group.PATCH("/accepted/:paymail", action.oldAccept)
+		group.PATCH("/rejected/:paymail", action.oldReject)
+		group.PATCH("/confirmed/:paymail", action.oldConfirm)
+		group.PATCH("/unconfirmed/:paymail", action.oldUnconfirm)
 
 		group.POST("search", action.search)
 	})
 
 	return apiEndpoints
 }
+
+// NewHandler creates the specific package routes
+func NewHandler(appConfig *config.AppConfig, services *config.AppServices) (routes.APIEndpointsFunc, routes.APIEndpointsFunc) {
+	action := &Action{actions.Action{AppConfig: appConfig, Services: services}}
+
+	contactsAPIEndpoints := routes.APIEndpointsFunc(func(router *gin.RouterGroup) {
+		group := router.Group("/contacts")
```

Comment:
There is no DELETE contact endpoint, which seem to be reasonable operation (and is listed in JIRA task)

Reply:
I've created an implementation of the DELETE contact endpoint

Reply:
but I'm not sure if you can find a contact ID from the paymail in a simpler way, that does not require passing an empty JSON or skipping the `Content-Type: application/json` header for the endpoint

Reply:
I've changed it - it required changes in the engine, but now works as it should